HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentView.swift

```diff
@@ -68,31 +73,32 @@ final class PaymentComponentView: UIView {
         let image = UIImageNamedPreferred(named: viewModel.moreInformationIconName)
         button.setImage(image, for: .normal)
         button.tintColor = viewModel.moreInformationAccentColor
+        button.addTarget(self, action: #selector(tapOnMoreInformationButtonAction), for: .touchUpInside)
         return button
     }()
     
     private lazy var selectBankView: UIView = {
         let view = UIView()
         view.translatesAutoresizingMaskIntoConstraints = false
-        view.frame = CGRect(x: 0, y: 0, width: .max, height: 185)
+        view.frame = CGRect(x: 0, y: 0, width: .greatestFiniteMagnitude, height: Constants.bankViewHeight)
         return view
     }()
     
-    private lazy var selectBankLabel: UILabel = {
+    private lazy var selectYourBankLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.text = viewModel.selectBankLabelText
-        label.textColor = viewModel.selectBankAccentColor
-        label.font = viewModel.selectBankLabelFont
+        label.text = viewModel.selectYourBankLabelText
+        label.textColor = viewModel.selectYourBankAccentColor
+        label.font = viewModel.selectYourBankLabelFont
         label.numberOfLines = 0
         return label
     }()
     
     private lazy var selectBankPickerView: UIView = {
         let view = UIView()
         view.translatesAutoresizingMaskIntoConstraints = false
-        view.frame = CGRect(x: 0, y: 0, width: .max, height: 56)
-        view.layer.cornerRadius = 12
+        view.frame = CGRect(x: 0, y: 0, width: .greatestFiniteMagnitude, height: Constants.selectBankViewHeight)
+        view.layer.cornerRadius = Constants.cornerRadius
         view.layer.borderWidth = 1
```

Comment:
[Suggestion]: @razvancapra could you please also move `borderWidth` in the Constants?